src/imitation/algorithms/bc.py

```diff
@@ -116,8 +128,61 @@ def set_expert_dataset(
         assert expert_dataset.size() is not None
         self.expert_dataset = expert_dataset
 
+    def _build_tf_graph(self):
+        with tf.variable_scope("bc_supervised_loss"):
+            with tf.variable_scope("model") as model_scope:
+                self.policy_kwargs = dict(
```

Comment:
Should these be configurable?

Reply:
This is fixed in a later commit, thanks for pointing this out